### ISSUE TITLE
Pod structure makes x-toggle undiscoverable for some apps

### ISSUE THREAD

**Issue by ef4** [knownasilya/ember-cli-toggle#40]
If an app customizes `podModulePrefix`, their resolver won't find `app/components/x-toggle/component.js`.

I think the only way for addons to remain 100% compatible is to stick with the stock structure (app/components/x-toggle.js).

**Comment by ef4** [comment]
It's possible to workaround this within an app by creating `app/components/x-toggle.js` explicitly and doing the reexport:

```js
export { default } from 'ember-cli-toggle/components/x-toggle/component';
```

**Comment by knownasilya** [comment]
Already fixed with 0.6.1https://github.com/knownasilya/ember-cli-toggle/commit/3705bcc59a73aa00b406849c40ad7ec27ac30158 see the 0.x branch.
